CorsixTH/Lua/dialogs/edit_room.lua

```diff
@@ -1257,10 +1257,10 @@ function UIEditRoom:placeWindowBlueprint()
   end
 end
 
-function UIEditRoom:setWindowBlueprint(x, y, wall)
-  local orig_x = x
-  local orig_y = y
-  local orig_wall = wall
+function UIEditRoom:setWindowBlueprint(orig_x, orig_y, orig_wall)
+  local x = orig_x
```

Comment:
And yet they get written to on Line 1286 anyway. But that's ok, closer to const parameters.